subsys/bluetooth/host/hci_core.c

```diff
@@ -4108,6 +4108,14 @@ int bt_enable(bt_ready_cb_t cb)
 	k_thread_name_set(&bt_workq.thread, "BT RX");
 #endif
 
```

Comment:
@lylezhu2012, I noticed that this `init` function is called just before the `open`, wouldn’t be possible to perform `init` actions within `open` ?

Reply:
updated